docs/table-partitioning-bucketing-sort-order-indexes.md

```diff
@@ -285,3 +285,136 @@ tableCatalog.createTable(
 
 </TabItem>
 </Tabs>
+
+## Indexes
+
+To define an indexed table, you should utilize the following three components to construct a valid indexed table.
+
+- IndexType. Represents the type of index, such as primary key or unique key.
+
+| IndexType     | Description                                                                                                                                       | JSON            | Java                      |
+|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|---------------------------|
+| PRIMARY_KEY   | In a relational database is a field or a combination of fields that uniquely identifies each record in a table.                                   | `PRIMARY_KEY`   | `IndexType.PRIMARY_KEY`   |
+| UNIQUE_KEY    | In a relational database is a field or a combination of fields that ensures each record in a table has a distinct value or combination of values. | `UNIQUE_KEY`    | `IndexType.UNIQUE_KEY`    |
+
+- Name. It defines the name of the index.
```

Comment:
Please add more detailed about it, for example, `The name of `PRIMARY_KEY` must be `PRIMARY` and ...